<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlManual</class>
 <widget class="QDialog" name="ControlManual">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Manual</string>
  </property>
  <widget class="QPushButton" name="btnMasX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMenosX">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGirarModulo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>350</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 16px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>Girar Modulo 
Turn Module</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLlevarOrigen">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 16px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>Llevar al Origen 
Go to Setpoint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPruebaManual">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 16px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>Prueba Manual 
Manual Test</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtEjeX">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignCenter';</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtEjeY">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignCenter';</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtEjeZ">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">qproperty-alignment: 'AlignCenter';</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCamara">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>310</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon_camara.jpg</normaloff>icon_camara.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVolver">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 16px;
border-radius: 8px;
padding: 8px 28px;
border: none;
text-align: left;</string>
   </property>
   <property name="text">
    <string>Volver 
Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMasY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMasZ">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMenosY">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMenosZ">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="txtFiltros">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20px;
qproperty-alignment: 'AlignCenter';</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFiltros_D">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: rgb(255, 255, 255);
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>flecha_D.png</normaloff>flecha_D.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lblCamara">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>431</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFiltros_I">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: white;
font-weight: bold;
font-size: 18px;
border-radius: 8px;
padding: 12px 0;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>flecha_I.png</normaloff>flecha_I.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
